packages/patternfly-3/patternfly-react/sass/patternfly-react/_date-time-picker.scss

```diff
@@ -0,0 +1,22 @@
+.input-group.date-time-picker-pf.input-group {
+  width: 41.66666667%;
+}
+
+.bootstrap-datetimepicker-widget.dropdown-menu.top {
+  max-width: none;
+  margin-top: -10px;
+  .arrow {
+    left: 10% !important;
+  }
+  .popover-content {
+    color: $color-pf-black-900;
+  }
+  .clock-btn {
+    border: 0;
+    box-shadow: none;
+    color: #363636;
+    display: block;
+    padding-bottom: 4px;
+    padding-top: 4px;
+  }
+}
```

Comment:
I saw that there are small places where the elements are a bit off position, but I guess that the designers will have more to say about it than myself:

**Current implementation:**

![date-picker](https://user-images.githubusercontent.com/26363699/53699722-124f1d00-3df4-11e9-99fc-74d3adee5d27.png)

**Original Design:**

![date-picker-design](https://user-images.githubusercontent.com/26363699/53699724-1b3fee80-3df4-11e9-95c4-0322627758ef.png)